packages/reactotron-server/src/server/datastore/commands.ts

```diff
@@ -7,8 +7,22 @@ export class Commands {
     this.commands.push(command)
   }
 
-  byClientId(clientId) {
-    return this.commands.filter(cmd => cmd.clientId === clientId)
+  filterItem(item: Command, clientId?: string, filter?: [string]) {
```

Comment:
With this function it becomes a little more then just a "store" because it offers the ability to run common filtering tasks against and arbitrary command. We should keep an eye on this and if this type of thing starts growing consider taking this (and the composable) function out of here and put it into a utility file or something.